cartography/intel/aws/ec2/launch_templates.py

```diff
@@ -22,7 +22,19 @@ def get_launch_templates(boto3_session: boto3.session.Session, region: str) -> l
     paginator = client.get_paginator('describe_launch_templates')
     templates: list[dict[str, Any]] = []
     for page in paginator.paginate():
-        templates.extend(page['LaunchTemplates'])
+        paginated_templates = page['LaunchTemplates']
+        for template in paginated_templates:
+            try:
+                template_versions = get_launch_template_versions_by_template(boto3_session, template, region)
+            except Exception as e:
+                logger.warning(
+                    f"Failed to get launch template versions for {template['LaunchTemplateId']}: {e}",
+                    exc_info=True,
+                )
+                template_versions = []
+            # Using a key not defined in latest boto3 documentation
+            template['GetVersions'] = template_versions
```

Comment:
Rather than tack on this in a field on the template, how about we let `get_launch_templates` return a tuple of 2 things: 

1. the list of templates
2. the list of template versions

I realize that the original code did it this way, but I think it's cleaner to have multiple return values in this case. There's no need to have the version data embedded in the template dict itself. 

By doing this we would also no longer need the `get_launch_template_versions` below (the one that just goes through each template and grabs each version and puts it in a list).